### PR TITLE
Run Python services using volumes. Add `local` make target

### PR DESCRIPTION
**Pull request by JamesGardiner**
# Motivation and Context
Allows Python services to be run from a mounted Volume in docker containers. This allows hot-reloading of code-changes, creating a tighter feedback cycle for developers.

# What has changed
- Added a new `ras-local.yml` docker-compose file.
- Added the `make` target `local` that will start all RAS Python services mounted under a Volume using docker-compose.

# How to test?
- Use `make local` to start the local Python services.
- Make sure all the relevant repos are up-to-date with latest Master on Github
- Run the acceptance tests against the environment, ensuring they pass
- Make arbitrary changes to the codebase to check that hot-reloading is on

# Links
https://trello.com/c/4O9axOVO/155-mount-python-services-using-volumes-in-docker-containers-8

### CODE REVIEW

File: ras-local.yml [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+      test: ["CMD", "curl", "-f", "http://localhost:8040/info"]
+      interval: 1m30s
+      timeout: 10s
+      retries: 3
+
+  response-operations-ui:
+    container_name: response-operations-ui
+    build: ${RAS_HOME}/response-operations-ui
+    volumes:
+      - ${RAS_HOME}/response-operations-ui:/response-operations-ui
```
Comment (jcox-dev) [inline ras-local.yml]: `${RAS_HOME}/response-operations-ui:/response-operations-ui` -> `${RAS_HOME}/response-operations-ui:/response_operations_ui` 

([underscores](https://github.com/ONSdigital/response-operations-ui/blob/9d20aecdefd8ec26b89fc652cf61b2d86464caeb/Dockerfile#L4))

**Comment by benjefferies** [comment]
I've stole an modified a neat little script to change all repos to master and pull if you need to do it for all repos https://gist.github.com/benjefferies/f95a4f0d452805c7174e630848cc8290

File: Makefile [hunk 2]
```diff
@@ -3,8 +3,11 @@ up:
 	pipenv install --dev
 	pipenv run python setup_database.py
 
+local:
+	docker-compose -f ras-local.yml up -d
```
[the remark this answers is not in the document]
Comment (benjefferies) [reply Makefile]: The documentation @JamesGardiner added suggests we can do this 😄. I think it's worth adding

File: ras-local.yml (hunk 1 above)
Comment (benjefferies) [inline ras-local.yml]: Why not /app for consistency?

**Comment by JamesGardiner** [comment]
Thanks @benjefferies and @jcox-dev

**Pull request merged**